### ISSUE TITLE
C210 Firmware 1.3.9 issue

### ISSUE THREAD

**Issue by mitzu2250** [JurajNyiri/HomeAssistant-Tapo-Control#437]
### Description

Since upgrading the firmware of my C210 camera to version 1.3.9 it stopped working in HA.

I'm using the latest version of the integration - 5.4.0.
Same version works on 1.3.7 linked to the same system.

Tried rebooting camera, HA instance (both Restart Home Assistant and Reboot system).

Looking at logs I see:
![image](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/assets/99750119/f27ee029-dfd5-4dab-8a48-f2ace2e3fe67)

Login data is 100% correct.
(https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/files/13259537/home-assistant_2023-11-05T12-08-22.782Z.log)

Detailed log: [home-assistant_2023-11-05T12-08-22.782Z.log]

Tried deleting and setting up again, it fails on the cloud password step:
![image](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/assets/99750119/0bd24dca-de22-4738-97e1-485dc03ffba5)

Log with setup try: 
[home-assistant_2023-11-05T12-17-58.711Z.log](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/files/13259545/home-assistant_2023-11-05T12-17-58.711Z.log)

### Reproduction Steps

1. Update camera to 1.3.9 or higher
2. Try to view the camera feed
3. Doesn't work - integration fails

### Expected behavior

Camera should work normally, entities should be visible.

### If applicable, add error logs.

[home-assistant_2023-11-05T12-08-22.782Z.log](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/files/13259546/home-assistant_2023-11-05T12-08-22.782Z.log)
[home-assistant_2023-11-05T12-17-58.711Z.log](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/files/13259547/home-assistant_2023-11-05T12-17-58.711Z.log)

### Device Firmware

1.3.9 Build 231019 Rel.13805n(4555)

### Using stream component

Yes

### Does camera work via official integrations?

N/A

### Camera has all attributes filled out in developer tools

No

### HASS Environment

VM

### Search for similar issues

Yes

**Comment by JurajNyiri** [comment]
Duplicate of https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/436 .

Read through the thread there, especially the last messages.

If you changed cloud password, unblock it in firewall, factory reset camera, remove from account, add back, try again via integration.

If you are still experiencing issues:

https://github.com/JurajNyiri/pytapo/tree/debug_request

Download it manually, and next to README.md create file test.py with content:
```
from pytapo import Tapo

# Camera
user = ""  # 3rd party account name
password_cloud = ""  # cloud password
host = ""  # ip of the camera, example: 192.168.1.52

tapo = Tapo(host, "admin", password_cloud)
request = tapo.getBasicInfo()
print(request)
```

Post the whole output of the script, it should already have everything important redacted.
